### ISSUE TITLE
config.conf - color settings in layout section do not accept hex or rgb values

### ISSUE THREAD

**Issue by ctreffe** [ctreffe/alfred#270]
Currently, the documentation states that the _header_color_ and the _background_color_ settings in config.conf accept RGB, Hex, and CSS color keywords. However, the # prefix that seems to be necessary to enter a valid value can not be escaped in the config.conf.

Hence, users can currently only enter invalid values, which are correctly set in the css configuration (I checked). This issue exists both for local and online experiments on mortimer.

**Comment by jobrachem** [comment]
Oh, that's annoying. Does it work, if you put the hex code in a string? Like `"#0000FF"` instead of `#0000FF` ?

**Comment by ctreffe** [comment]
No, I tried many different workarounds to no avail. I think this setting is auto-imported and might need a customized import from the config.conf (I think something like this was possible in older alfred versions), or maybe there is a better solution to this issue, for example conditionally adding the "#" to the imported string (when it is not a word like "blue")?

**Comment by ctreffe** [comment]
On further investigation and after consulting ChatGPT, a simple workaround has been found:
The # prefix will not be interpreted as an inline comment if it is not preceded by whitespace. Therefore, the following line in config.conf works as expected:

`header_color =#ff9900  # Color of the header bar (RGB, Hex, CSS color keywords)`
